plugins/analytics-module-ga/README.md

```diff
@@ -0,0 +1,144 @@
+# Analytics Module: Google Analytics
+
+This plugin provides an opinionated implementation of the Backstage Analytics
+API for Google Analytics. Once installed and configured, analytics events will
+be sent to GA as your users navigate and use your Backstage instance.
+
+This plugin contains no other functionality.
+
+## Installation
+
+1. Install this plugin in your Backstage app:
+   `cd packages/app && yarn add @backstage/plugin-analytics-module-ga`
+2. Add the API implementation to your App and configure the plugin (see below).
+
+```tsx
+// packages/app/src/apis.ts
+import { analyticsApiRef, configApiRef } from '@backstage/core-plugin-api';
+import { GoogleAnalytics } from '@backstage/plugin-analytics-module-ga';
+
+export const apis: AnyApiFactory[] = [
+  // Instantiate and register the GA Analytics API Implementation.
+  createApiFactory({
+    api: analyticsApiRef,
+    deps: { configApi: configApiRef },
+    factory: ({ configApi }) => GoogleAnalytics.fromConfig(configApi),
+  }),
+];
+```
+
+## Configuration
+
+The following is the minimum configuration required to start sending analytics
+events to GA. All that's needed is your Universal Analytics tracking ID:
+
+```yaml
+# app-config.yaml
+app:
+  analytics:
+    provider: ga
+    ga:
+      trackingId: UA-0000000-0
```

Comment:
Is it possible to have different tracking ids, for example, one for production and the other for development?

Reply:
Example:
https://github.com/react-ga/react-ga#reactgainitializegatrackingid-options

![image](https://user-images.githubusercontent.com/6290749/134520697-49392cdf-3795-4585-968d-4db0c434b498.png)

Reply:
I mentioned elsewhere in this PR, but I recommend we start with the simple, single tracking ID config, and if someone needs multiple trackers, we can add that in later.